src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -252,7 +252,15 @@ describe('StripeV3PaymentStrategy', () => {
                 expect(paymentActionCreator.submitPayment).toHaveBeenCalledWith(
                     expect.objectContaining({
                         paymentData: expect.objectContaining({
-                            shouldSetAsDefaultInstrument: true,
```

Comment:
is `shouldSetAsDefaultInstrument` still supported? Could this break checkouts using the older version of the SDK>

Reply:
I kept the consistency and remove this flag out of formattedPaylaod